test/puppetlabs/jdbc_util/Messages_eo.properties

```diff
@@ -0,0 +1 @@
+Initialization\ resulted\ in\ an\ error\:\ {0}=This_is_a_translated_string: {0}
```

Comment:
Property files are a little magic.  I'll add it and make sure it doesn't blow up.  If it does, I'll remove it. :)